Anima/math-tools/special_functions/animaKummerFunctions.h

```diff
@@ -48,7 +48,9 @@ class KummerIntegrand
     
     double operator() (const double t)
     {
-        return std::exp(m_XValue * (t - (double)(m_XValue > 0))) * std::pow(t, m_AValue - 1.0) * std::pow(1.0 - t, m_BValue - m_AValue - 1.0);
+        // For better numerical stability
+        double tModified = t - (m_XValue > 0);
```

Comment:
Same here. Is `(m_XValue > 0)` cast to double ? If not necessarily, might be sager to add `(double)(m_XValue > 0)`.

Reply:
Same as before, yes it is cast to double, no worries.